src/Uno.UI.Tests/Windows_Globalization/Given_Rounder.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Globalization.NumberFormatting;
+
+namespace Uno.UI.Tests.Windows_Globalization
+{
+	[TestClass]
+    public class Given_Rounder
+    {
+
+        [DataTestMethod]
+        [DataRow(4.48, 0, 4d)]
+        public void When_UsingRoundMethod(double value, int digits, double expected)
+        {
+            var rounded = Rounder.Round(value, digits, RoundingAlgorithm.RoundHalfAwayFromZero);
+            Assert.AreEqual(expected, rounded);
+        }
+
+        [DataTestMethod]
+        [DataRow(1.5, true)]
+        [DataRow(0.5, true)]
+        [DataRow(-0.5, true)]
+        [DataRow(-12.5, true)]
+        [DataRow(-4.5, true)]
+        [DataRow(-4.50001, false)]
+        public void When_Fraction_Is_Half_Then_Return_True(double value, bool expected)
+        {
+            Assert.AreEqual(expected, Rounder.IsFractionExactlyHalf(value));
+        }
+
+		[DataTestMethod]
```

Comment:
The indentation is wrong for the remainder of the file, try format document in VS to adjust it 👍

Reply:
The problem was because of using spaces for indentation not Tabs.